src/pages/student_council.tsx

```diff
@@ -0,0 +1,185 @@
+                <Grid.Col span={{ base: 12, md: 6 }}>
+                    <Paper className="flex flex-col rounded-lg p-6 md:h-96 md:flex-row">
+                        <div>
+                            <h1 className="text-xl md:text-4xl">About</h1>
+                            <p className="w-72 text-lg">
+                                The Student Council is the official student body of Stamford International University.
+                                The council is responsible for representing the student body, organizing events, and
+                                communicating with the university&apos;s administration.
+                            </p>
+                        </div>
+                        <Image
+                            className="rounded-md md:ml-auto"
+                            src="/assets/images/student_council/group.jpg"
+                            alt="Student Council Group Photo"
+                            width={3641 / 11}
+                            height={3545 / 11}
+                        />
+                    </Paper>
+                </Grid.Col>
+                <Grid.Col span={{ base: 12, md: 6 }}>
+                    <Paper className="flex flex-col rounded-lg p-6 md:h-96 md:flex-row">
+                        <div>
+                            <h1 className="text-xl md:text-4xl">Upcoming Events</h1>
+                            <p className="w-96 text-base">
+                                The &quot;Welcome Freshmen&quot; event, orchestrated by the Student Council at Stamford
+                                International University, symbolizes the warm embrace awaiting new students. This event
+                                serves as an introduction to the vibrant community, showcasing the Council&rsquo;s
+                                dedication to fostering a welcoming environment and facilitating a smooth transition
+                                into university life.
+                            </p>
+                        </div>
+                        <Image
+                            className="rounded-md md:ml-auto"
+                            src="/assets/images/student_council/example.png"
+                            alt="Example Event Poster"
+                            width={3456 / 14}
+                            height={4608 / 14}
+                        />
+                    </Paper>
+                </Grid.Col>
+            </Grid>
+
+            <div className="pt-20">
+                <Carousel withIndicators
+                    slideSize="25%" height={200} slideGap="md" align="start" >
+                    {carouselSlides.map((slide, index) => (
+                        <Carousel.Slide key={index}>
+                            <Image src={slide.src} alt={slide.alt} className="w-full h-full object-cover" />
+                        </Carousel.Slide>
+                    ))}
+                </Carousel>
+            </div>
```

Comment:
Couldn't you use the carousel component @chinathaip made?
https://github.com/stamford-syntax-club/stamfordcenter-frontend/blob/main/src/components/core/ImageSlideShow.tsx

Reply:
Rebase onto main btw